freetypefont.mod/freetypefont.bmx

```diff
@@ -200,18 +200,25 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 				MemFree buf
 				Return Null
 			EndIf
-		ElseIf String(src).Find( "::" )>0
-			Local tmp:Byte[]=LoadByteArray( src )
-			buf_size=tmp.length
-			If Not buf_size Return Null
-			buf=MemAlloc( Size_T(buf_size) )
-			MemCopy buf,tmp,Size_T(buf_size)
-			If FT_New_Memory_Face( ft_lib,buf,buf_size,0,Varptr ft_face )
-				MemFree buf
-				Return Null
-			EndIf
+		ElseIf Not String(src) 'empty tring

```

Comment:
Heh.. I'd push that down to the bottom as an
```
Else
    Return Null
```
And have the other be `Else If String(src)`

I think it's just a bit easier to read.

Reply:
Right. That is possible.

I am just "afraid" that it might be better if you just copied what is useful and created your own commit - saves time :-)